README.md

```diff
@@ -117,7 +117,7 @@ arbitrary LLVM version.
 
        $ cd <triton install>/python
```

Comment:
Can you change this line to
cd <triton install>
since later python directory is specified to `pip install -e`

Reply:
Sounds good. BTW, do you know why some tests are failing? It doesn't seem they can be affected by this patch.

Reply:
the failing test seem to be noise in the perf runs. This should go away on the next run.

Reply:
Looks like the test failed again after a retry.